manifests/helm-profiles/remote.yaml

```diff
@@ -0,0 +1,9 @@
+# The remote profile enables installing istio with a remote control plane. The `base` and `istio-discovery` charts must be deployed with this profile.
+istiodRemote:
+  enabled: true
+configMap: false
+telemetry:
```

Comment:
Wondering if for the remote profile we want to default to configCluster = true? I'd imagine in most cases someone uses the istiod-remote, it would usually be for a config cluster. I think setups where you have multiple remotes and non-config clusters attached to a primary Istiod would be rare or more complex cases: https://istio.io/latest/docs/setup/install/external-controlplane/#adding-clusters

Reply:
> Wondering if for the remote profile we want to default to configCluster = true?  I'd imagine in most cases someone uses the istiod-remote, it would usually be for a config cluster. I think setups where you have multiple remotes and non-config clusters attached to a primary Istiod would be rare or more complex cases: https://istio.io/latest/docs/setup/install/external-controlplane/#adding-clusters

1. Yeah, we probably should.
2. Current default is `false`
2. We have some integ tests that assume it is `false`.

I'm not against changing this in a followup to this PR, but I am trying to be minimally invasive WRT the existing integration tests around this stuff in this PR, because they are really the only thing describing the desired behavior ATM.